### PR TITLE
Some bugs fixes for base images

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

**Comment by lorenzo-stoakes** [comment]
Confirmed fixes #137.

File: node/generate-stackbrew-library.sh [hunk 1]
```diff
@@ -61,8 +61,7 @@ aliases=(
 	[6.2.2]='6 latest'
 	[5.11.1]='5'
 	[4.4.5]='4'
-	[0.12.14]='0.12 0'
-	[0.10.45]='0.10'
+	[0.12.14]='0'
```
Comment (lorenzo-stoakes) [inline node/generate-stackbrew-library.sh]: I'm not 100% clear on what this is used for, but is it necessary to specifically alias 0.12.14 to '0'? As 0.12.15 is the most recent version, would we specify that anywhere?

Comment (nghiant2710) [inline node/generate-stackbrew-library.sh]: We've not supported that version yet. 0.12.14 is the current latest 0.12.x

Comment (lorenzo-stoakes) [inline node/generate-stackbrew-library.sh]: Ok, in that case why do we alias it at all @nghiant2710 ?

Comment (nghiant2710) [inline node/generate-stackbrew-library.sh]: To make a tag with major version only that points to latest version.

Comment (lorenzo-stoakes) [inline node/generate-stackbrew-library.sh]: Ah ok I see

**Comment by lorenzo-stoakes** [comment]
LGTM

**Comment by lorenzo-stoakes** [comment]
(Am assuming you've tested the other components I couldn't test @nghiant2710 )

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**